stim/prompt.go

```diff
@@ -1,10 +1,36 @@
 package stim
 
 import (
-	"github.com/manifoldco/promptui"
+	"os"
 	"strings"
+
+	"github.com/chzyer/readline"
+	"github.com/manifoldco/promptui"
 )
 
+// stderr implements an io.WriteCloser that skips the terminal bell character
+// (ASCII code 7), and writes the rest to os.Stderr. It's used to replace
+// readline.Stdout, that is the package used by promptui to display the prompts.
+type stderr struct{}
```

Comment:
I would rename the struct to something more specific like noBellStderr

Reply:
Sure, I can change that.